app/views/RoomView/index.js

```diff
@@ -783,6 +817,16 @@ class RoomView extends React.Component {
 		return message;
 	}
 
+	renderScrollButton = () => {
```

Comment:
After the changes, it's going to look like this:
<img src="https://user-images.githubusercontent.com/804994/74954344-ba813b80-53e1-11ea-8b47-6585f20c8b4d.png" width="350" />
<img src="https://user-images.githubusercontent.com/804994/74954397-cff66580-53e1-11ea-91f6-e2882303df08.png" width="350" />
<img src="https://user-images.githubusercontent.com/804994/74954419-da186400-53e1-11ea-843c-a0e8aafe2def.png" width="350" />